lib/setup.php

```diff
@@ -526,6 +526,22 @@ function handle_wholesaler_message() {
   }
 } );
 
+/**
+ * Set cron for increasing fake message number
+ */
+if ( ! wp_next_scheduled( 'increase_fake_message_number' ) ) {
+  wp_schedule_event( time(), 'hourly', 'increase_fake_message_number' );
+}
+add_action( 'increase_fake_message_number', function() {
+	$options = get_fields('options');
+  if ( ! isset( $options['fake_message_number'] ) || ! isset( $options['fake_message_max_increase_constant'] ) ) return;
+  $increse_constant = rand( 0, (int) $options['fake_message_max_increase_constant'] ); // Generate random number from 0 to user defined value
```

Comment:
Ne že by to něčemu vadilo, ale asi by se to mělo jmenovat increase a ne increse

Reply:
Díky, opraveno :)